packages/cardano-services/src/Program/options/providers.ts

```diff
@@ -0,0 +1,79 @@
+import { newOption } from './util';
+
+export enum ProviderImplementation {
+  TYPEORM = 'typeorm',
+  BLOCKFROST = 'blockfrost',
+  DBSYNC = 'dbsync',
+  // Below ones are specific to TxSubmitProvider
+  'SUBMIT_NODE' = 'submit-node',
```

Comment:
Is there a difference? :thinking: 
```suggestion
  SUBMIT_NODE = 'submit-node',
```